backend/program/scrapers/jackett.py

```diff
@@ -163,19 +170,21 @@ def _search_movie_indexer(self, item: MediaItem, indexer: JackettIndexer) -> Lis
             "cat": "2000",
             "q": item.title,
         }
-        if indexer.movie_search_capabilities and "year" in indexer.movie_search_capabilities:
-            if hasattr(item.aired_at, "year") and item.aired_at.year: params["year"] = item.aired_at.year
+        year = None
+        if hasattr(item.aired_at, "year") and item.aired_at.year: 
+            year = item.aired_at.year
+        
         if indexer.movie_search_capabilities and "imdbid" in indexer.movie_search_capabilities:
             params["imdbid"] = item.imdb_id
 
         url = f"{self.settings.url}/api/v2.0/indexers/{indexer.id}/results/torznab/api"
-        return self._fetch_results(url, params, indexer.title, "movie")
+        return self._fetch_results(url, params, indexer.title, "movie", year)
 
     def _search_series_indexer(self, item: MediaItem, indexer: JackettIndexer) -> List[Tuple[str, str]]:
         """Search for series on the given indexer"""
         if indexer.tv_search_capabilities == None:
             return []
-        q, season, ep = self._get_series_search_params(item)
+        q, year, season, ep = self._get_series_search_params(item)

```

Comment:
I'm super reluctant to use year queries with show scraping as its normal for shows to not have years in their torrent names.. Movies usually do, but its fairly uncommon for shows to have a year in the title

Reply:
Yeah I get what you mean; what about a new config, leave it up to the user ? 

I tried with using a year param but that caused so many lost results so my solution was this to maximize results and possible torrents from information we have on hand.

Reply:
we could check if there are like.. 2-5+ results that match the year of the show we are looking for, then use the year results instead.. otherwise, use the results without a year "param" 🤔

Reply:
Reason I decided on regex and using pub year was to search the index with the title getting max possible results. Both work however my goal is to try to not lose any results that I can possibly give to debrid.

Then manually working it the normally small part off junk seems like 95% off the time torrents aren't getting removed according to the logs, vs sending in the year param which forces a year to be in the title. 

The check aim to determine if a torrent is valid based on the torrent release vs show release. Then if there is a year in the torrent check if any off the year's mentioned are the show year. (2015-2017), or (2015, 2016, 2017)